app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3 and command.startswith("cp"):
```

Comment:
But for example if command will equal to "cpcpcpcpcpcppcpc file.txt file2.txt"
It will return true. Due to task command must be "cp file.txt file2.txt"